src/lib.rs

```diff
@@ -34,6 +34,7 @@ pub mod data_tree;
 pub mod fs_tree_builder;
 pub mod get_size;
 pub mod json_data;
+pub mod mount_point;
```

Comment:
Since `mount_point` should not be `pub`. This module should be moved to `src/app/sub/mount_point.rs`.